BuildingSync.xsd

```diff
@@ -14155,9 +14155,11 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
       <xs:restriction base="xs:string">
         <xs:enumeration value="Property management company"/>
         <xs:enumeration value="Corporation/partnership/LLC"/>
-        <xs:enumeration value="Religious organization"/>
+        <xs:enumeration value="Privately owned"/>
         <xs:enumeration value="Individual"/>
         <xs:enumeration value="Franchise"/>
+        <xs:enumeration value="Religious organization"/>
+        <xs:enumeration value="Non-profit organization"/>
```

Comment:
@markborkum -- I made your request simply `non-profit organization` since there wasn't any non-profit organization enum, so stating `other non-profit organization` made little sense.